apps/mds/docs/public/specs/event_ongoing_banner.html

```
@@ -0,0 +1,1230 @@
+    <table class="ref state-mini">
+      <thead>
+        <tr><th colspan="3"><strong>ended · 종료 (banner 사라짐)</strong> <small>hidden · 결과 공개 후 1주일 경과</small></th></tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td class="state-mini__mock" rowspan="6" style="background: var(--color-surface); color: var(--color-text-secondary); font-size: 12px; text-align: center; padding: 32px;">
+            <em>banner not rendered</em><br><br>
+            (이 phase는 hidden state — MyTicketsPage에서 자동으로 사라지고, 회고는 <a href="/specs/purchase_history_page.html">PurchaseHistory</a>에서 timeline으로 확인)
+          </td>
+          <td class="state-mini__aspect">조건</td>
+          매칭 결과 공개 후 또는 noShow 시점부터 <strong>1주일(7일)</strong> 경과 시 자동으로 ended로 전환. results / noShow 모두 동일한 7일 정책. 7일 초과 시점에 banner는 MyTicketsPage list에서 사라지고, PurchaseHistory 지난 티켓 timeline으로 이관 (재진입 가능).</td>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </td> ]

(tag-pair)